src/PowerShellEditorServices.Protocol/MessageProtocol/Channel/NamedPipeServerChannel.cs

```diff
@@ -11,35 +11,47 @@ namespace Microsoft.PowerShell.EditorServices.Protocol.MessageProtocol.Channel
     public class NamedPipeServerChannel : ChannelBase
     {
         private ILogger logger;
-        private NamedPipeServerStream pipeServer;
+        private NamedPipeServerStream inOutPipeServer;
+        private NamedPipeServerStream outPipeServer;
 
         public NamedPipeServerChannel(
-            NamedPipeServerStream pipeServer,
+            NamedPipeServerStream inOutPipeServer,
             ILogger logger)
         {
-            this.pipeServer = pipeServer;
+            this.inOutPipeServer = inOutPipeServer;
+            this.outPipeServer = null;
```

Comment:
nit: This line can be removed, the field will already be initialized with `null` by default.